src/schemas/projectFunder.ts

```diff
@@ -0,0 +1,92 @@
+import gql from "graphql-tag";
+
+export const typeDefs = gql`
+  extend type Query {
+    "Get all of the Users that a Funders to the research project"
+    projectFunders(projectId: Int!): [ProjectFunder]
+
+    "Get all of the Users that are Funders for the specific Plan"
+    planFunders(planId: Int!): [ProjectFunder]
+  }
+
+  extend type Mutation {
+    "Add a Funder to a research project"
+    addProjectFunder(input: AddProjectFunderInput!): ProjectFunder
+    "Update a Funder on the research project"
+    editProjectFunder(input: EditProjectFunderInput!): ProjectFunder
+    "Remove a research project Funder"
+    removeProjectFunder(projectFunderId: Int!): Boolean
+
+    "Add a Funder to a Plan"
+    addPlanFunder(planId: Int!, projectFunderId: Int!): PlanFunder
```

Comment:
L21: I think PlanFunder wasn't defined anywhere

Reply:
Its just a join table to projectFunders since we don't need need to make any changes, just reference the one(s) we want

Reply:
maybe makes sense to name the mutation differently? like `selectProjectFunder` or something?

Reply:
I'm seeing this error when I start up the app:
```
Attempting to connect to Redis"}
apollo-server   | Error: Unknown type "PlanFunder". Did you mean "Language"?
apollo-server   |     at assertValidSDL (/app/node_modules/graphql/validation/validate.js:135:11)
apollo-server   |     at buildASTSchema (/app/node_modules/graphql/utilities/buildASTSchema.js:44:34)
apollo-server   |     at makeExecutableSchema (/app/node_modules/@apollo/server/node_modules/@graphql-tools/schema/cjs/makeExecutableSchema.js:73:47)
apollo-server   |     at Function.constructSchema (/app/node_modules/@apollo/server/src/ApolloServer.ts:714:32)
apollo-server   |     at new ApolloServer (/app/node_modules/@apollo/server/src/ApolloServer.ts:274:37)
apollo-server   |     at Object.<anonymous> (/app/src/index.ts:28:22)
apollo-server   |     at Module._compile (node:internal/modules/cjs/loader:1480:14)
apollo-server   |     at Module._compile (/app/node_modules/source-map-support/source-map-support.js:521:25)
apollo-server   |     at Module.m._compile (/tmp/ts-node-dev-hook-06277461284618457.js:69:33)
apollo-server   |     at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
```

Reply:
Yeah, I just noticed that myself and pushed a fix.

Reply:
I renamed those mutations to:
```
    "Add a Funder to a Plan"
    selectProjectFunderForPlan(planId: Int!, projectFunderId: Int!): ProjectFunder

    "Remove a PlanFunder from a Plan"
    removeProjectFunderFromPlan(planId: Int!, projectFunderId: Int!): Boolean
```